--- a/src/Http/Requests/UpdateUserRequest.php
+++ b/src/Http/Requests/UpdateUserRequest.php
@@ -46,6 +46,6 @@
             'email' => [
                 'required',
                 'string',
                 'email',
                 'max:255',
-                Rule::unique('users', 'email')->ignore($this->route('id'))->whereNull('deleted_at'),
+                Rule::unique('users', 'email')->ignore($this->route('id'), 'uuid')->whereNull('deleted_at'),
             ],